modules/ROOT/pages/keyword-glossary.adoc

```diff
@@ -58,6 +58,14 @@ Typically used when modifying or importing large amounts of data.
 | Schema
 | Create a constraint that ensures all relationships with a particular type have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
 
+| xref::constraints/syntax.adoc#constraints-syntax-create-node-prop-type[CREATE CONSTRAINT [property_type\] [IF NOT EXISTS\] FOR (n:Label) REQUIRE n.property IS :: TYPE [OPTIONS {}\]]
```

Comment:
In a similar vein as above, I think we should be clearer with the user defined variables being distinct from the rest of the command.
```suggestion
| xref::constraints/syntax.adoc#constraints-syntax-create-node-prop-type[CREATE CONSTRAINT [constraint_name\] [IF NOT EXISTS\] FOR (n:Label) REQUIRE n.foo IS :: TYPE [OPTIONS {}\]]
```
This goes for all constraints, not just the new ones